### PR TITLE
Keybindings for other platforms in README.md

### PR DESCRIPTION
**Pull request by bekroogle**
Keybindings are in bullet lists, instead of tables as in atom/bookmarks#29

This really needs to exist somewhere. Being new to Atom, I only stumbled upon these bindings when browsing the repo, hoping to learn how to implement them.

### CODE REVIEW

**Comment by 50Wliu** [comment]
The image will also have to be updated for this.

**Comment by bekroogle** [comment]
How's this? (I just updated the README.md with the following screenshot.)

atom-bookmarks-screenshot

File: README.md [hunk 1]
```diff
@@ -2,10 +2,31 @@
 
 Bookmark lines in the editor.
 
+### Keybindings
+
```
Comment (50Wliu) [inline README.md]: :art: Extra newline here

**Comment by 50Wliu** [comment]
Other than my one small comment, could you please take the image in safe mode so that it matches the out-of-the-box experience?  Just so others don't get confused when they don't see the linter status :smile:.

**Comment by bekroogle** [comment]
Okay, I think I got the screenshot right and I removed the extraneous whitespace after "Keybindings".

// Sorry if I've made it a painful process--I haven't done many pull requests before.

**Comment by 50Wliu** [comment]
Thanks a lot for this, and don't worry about the process :).

**Comment by bekroogle** [comment]
My pleasure. It was a good learning experience for me and a nice diversion
from real work =]

**Comment by bekroogle** [comment]
@50Wliu, Do you have any idea how we could go about getting this updated README.md and keybindings to be reflected in the page at https://atom.io/packages/bookmarks?

Anyone who clicks on the bookmarks link in Settings > Packages gets sent to the atom.io/packages site, so someone would actually have to seek out this repo to find the keybindings for the other platforms.

**Comment by mnquintana** [comment]
@bekroogle I just pushed a new patch release for bookmarks, so https://atom.io/packages/bookmarks should be up-to-date now – sorry for the delay!

**Comment by 50Wliu** [comment]
Whoops, I forgot all about pushing a new release :grimacing:.  Sorry about that!

**Comment by bekroogle** [comment]
No worries. I only noticed because I hadn't used Atom in a while and
couldn't remember the Bookmarks keystrokes, so I clicked the link in the
Packages section to be reminded. :)

**Pull request merged**